### PR TITLE
Add dpkg -Qk

### PR DESCRIPTION
**Pull request by mondeja**

### CODE REVIEW

File: lib/zz_main.sh [hunk 1]
```diff
@@ -227,7 +227,7 @@ while :; do
       fi
       ;;
 
-    w|v)
+    w|v|k)
```
Comment (mondeja) [inline lib/zz_main.sh:230]: I'm not really sure if this should be here :think:

File: lib/dpkg.sh [hunk 2]
```diff
@@ -44,6 +44,14 @@ dpkg_Qe() {
   apt-mark showmanual "$@"
 }
 
+dpkg_Qk() {
+  if ! which debsums > /dev/null 2>&1; then
```
Comment (icy) [inline lib/dpkg.sh]: can you please replace with `command -v debsums`? It's more portable.

File: tests/dpkg.txt [hunk 3]
```diff
@@ -105,6 +105,17 @@ in ! echo Y | pacman -R screen
 in -Qi screen
 ou ^Status: deinstall
 
+# Verify package files
+in -Qk apt
+ou debsums binary does not exist
+in ! echo Y | pacman -S debsums
```
Comment (icy) [inline tests/dpkg.txt]: you can use `in -S --noconfirm debsums` too

**Review by icy — CHANGES_REQUESTED** [review]

File: lib/dpkg.sh (hunk 2 above)
Comment (icy) [reply lib/dpkg.sh]: of course the full command is `command -v debsums >/dev/null 2>&1`

**Comment by mondeja** [comment]
Thanks for your review, should be fine now

**Review by icy — APPROVED** [review]
Great, thanks a lot for your contribution.

**Review by icy — APPROVED** [review]

**Pull request merged**